api/src/pcapi/connectors/serialization/boost_serializers.py

```diff
@@ -56,12 +56,13 @@ def _convert_to_utc_datetime(datetime_with_tz_offset: datetime.datetime) -> date
     return datetime_with_tz_offset.astimezone(tz=datetime.timezone.utc)
 
 
-class ShowTime3(BaseModel):
+class ShowTime4(BaseModel):
```

Comment:
On pourrait ajouter  le même commentaire qu'au-dessus, car la raison de ce renommage est sinon très obscure:
`We transcribe their API schema and keep their name convention`